### PR TITLE
Add support for the Stripe-Account header

### PR DESCRIPTION
**Pull request by kyleconroy**
cc @bkrausz

### CODE REVIEW

**Comment by kyleconroy** [comment]
Alright, ready for review! r? @bkrausz

File: stripe/test/test_requestor.py [hunk 1]
```diff
@@ -290,6 +290,24 @@ def test_passes_api_version(self):
         self.check_call('get', headers=APIHeaderMatcher(
             extra={'Stripe-Version': 'fooversion'}, request_method='get'))
 
+    def test_uses_instance_account(self):
```
Comment (bkrausz) [inline stripe/test/test_requestor.py]: Maybe a test that these chain properly from an object to a related object?

**Comment by bkrausz** [comment]
Generally LGTM. Mild FUD that we're adding another property in the internals instead of building an object, but given Python's named-param-friendly world I can't think of too strong an argument against that.

File: stripe/test/test_requestor.py (hunk 1 above)
Comment (kyleconroy) [inline stripe/test/test_requestor.py]: Yep, will do.

**Comment by kyleconroy** [comment]
I actually started out using an object instead of another property, but I found that this ended up being more straight forward.

**Pull request merged**